Image Scrapper

```diff
@@ -0,0 +1,34 @@
+from bs4 import BeautifulSoup
+import requests
+import urllib.request
+
+from google.colab import drive
+drive.mount('/content/drive')
+
+import pandas as pd
+df = pd.read_csv('/content/drive/MyDrive/Ralph_Lauren_Retail_Feed.csv')
+print(df.shape)
+df.head()
+
+for i in range(len(df)):
+  print (i)
```

Comment:
We don't need prints, if you really want progress tracking [`tqdm`](https://github.com/tqdm/tqdm) is a neat library